build.sc

```diff
@@ -11,7 +11,7 @@ import mill.scalalib.api.ZincWorkerUtil.scalaNativeBinaryVersion
 import mill.scalalib.publish.{Developer, License, PomSettings, VersionControl}
 import scalalib._
 
-val millVersions                           = Seq("0.10.12", "0.11.1")
+val millVersions                           = Seq("0.10.0", "0.11.0") // scala-steward:off
```

Comment:
I think, we need at least Mill 0.10.6, as this version introduces the `semanticDbData` target which this plugins relies on.

```suggestion
val millVersions                           = Seq("0.10.6", "0.11.0") // scala-steward:off
```